packages/renderer/src/lib/kube/KubernetesDashboard.svelte

```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
+import { Link, NavPage } from '@podman-desktop/ui-svelte';
+
+import KubernetesCurrentContextConnectionBadge from '/@/lib/ui/KubernetesCurrentContextConnectionBadge.svelte';
+import {
+  kubernetesCurrentContextConfigMaps,
+  kubernetesCurrentContextDeployments,
+  kubernetesCurrentContextIngresses,
+  kubernetesCurrentContextNodes,
+  kubernetesCurrentContextPersistentVolumeClaims,
+  kubernetesCurrentContextRoutes,
+  kubernetesCurrentContextSecrets,
+  kubernetesCurrentContextServices,
+  kubernetesCurrentContextState,
+} from '/@/stores/kubernetes-contexts-state';
+import { NO_CURRENT_CONTEXT_ERROR } from '/@api/kubernetes-contexts-states';
+
+import { kubernetesContexts } from '../../stores/kubernetes-contexts';
+import ConfigMapSecretIcon from '../images/ConfigMapSecretIcon.svelte';
+import DeploymentIcon from '../images/DeploymentIcon.svelte';
+import IngressRouteIcon from '../images/IngressRouteIcon.svelte';
+import NodeIcon from '../images/NodeIcon.svelte';
+import PvcIcon from '../images/PVCIcon.svelte';
+import ServiceIcon from '../images/ServiceIcon.svelte';
+import KubernetesDashboardGuideCard from './KubernetesDashboardGuideCard.svelte';
+import KubernetesDashboardResourceCard from './KubernetesDashboardResourceCard.svelte';
+import KubernetesEmptyPage from './KubernetesEmptyPage.svelte';
+
+let noContexts = $derived($kubernetesCurrentContextState.error === NO_CURRENT_CONTEXT_ERROR);
+let currentContextName = $derived($kubernetesContexts.find(context => context.currentContext)?.name);
+
+let nodeCount = $derived($kubernetesCurrentContextNodes.length);
+let deploymentCount = $derived($kubernetesCurrentContextDeployments.length);
+let serviceCount = $derived($kubernetesCurrentContextServices.length);
+let ingressRouteCount = $derived($kubernetesCurrentContextIngresses.length + $kubernetesCurrentContextRoutes.length);
+let pvcCount = $derived($kubernetesCurrentContextPersistentVolumeClaims.length);
+let configMapSecretCount = $derived(
+  $kubernetesCurrentContextConfigMaps.length + $kubernetesCurrentContextSecrets.length,
+);
+
+function openKubernetesDocumentation() {
+  window.openExternal('https://podman-desktop.io/docs/kubernetes');
+}
```

Comment:
issue: missing return type

also openExternal is async so missing await or catch

Reply:
Fixed.